packages/react/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -46,6 +53,19 @@ export interface ToolbarItemProps
   /** Name or shorthand for Toolbar Item Icon */
   icon?: ShorthandValue
 
+  /** Shorthand for the submenu. */
+  menu?: ShorthandValue | ShorthandCollection
+
+  /** Indicates if the menu inside the item is open. */
+  menuOpen?: boolean
+
+  /**
+   * Event for request to change 'menuOpen' value.
+   * @param {SyntheticEvent} event - React's original SyntheticEvent.
+   * @param {object} data - All props and proposed value.
+   */
+  onMenuOpenChange?: ComponentEventHandler<ToolbarItemProps>
```

Comment:
Should we also hoist `onOpenChange` from `Popup` to here to have both on the same level?

Reply:
Let's skip it for now, we still have an access to it via `popup` shorthand